docs/source/integrations/ios-fairplay-sdk.md

```diff
@@ -1,138 +1,140 @@
-As an evolution of the the hugely successful VUPLAY SDK, this SDK has a number of key benefits: 
-● Complete control over the entire AVPlayer lifecycle 
-● Numerous optimisations to enable faster playback 
-● Bitcode support 
-● Support for building/linking against the simulators 
-The SDK is fully supported in both Objective-C and Swift applications. A demo application written in Swift is available on request. 
-Please contact support@vualto.com to request a download of the SDK.
-
-### REQUIREMENTS ###
-
-● Minimum iOS deployment target of 9.x or higher 
-● Xcode 8.3.3 
-● Swift 3.1 
-
-### XCODE INTEGRATION ###
-
-1. Copy the vudrmFairPlay.framework into your application's project directory 
-2. Navigate to your target and add vudrmFairPlay.framework to your ‘Embedded Binaries’. You should see that vudrmFairPlay.framework also now appears under ‘Linked Frameworks and Libraries’: 
-
-### INFORMATION ABOUT APPLICATION TRANSPORT SECURITY (ATS) ### 
-iOS/tvOS 9 introduces Application Transport Security (ATS) which restricts the use of insecure HTTP. 
-In order to permit playback of content over insecure HTTP exemptions need to be added into your application's Info.plist. For example to disable ATS for any connection you would add the following into your application's Info.plist: 
-<dict> 
-<key>NSAppTransportSecurity</key> <dict> 
+
+1.      Online Playback / Download and Offline Playback using a persisted token and license.
+    And legacy methods:
+2.      Make use of Apple’s AVPlayer and AVPlayerViewController to present users with the platform
+    default skins, with DRM content.
+3.      Create your own video player based on AVPlayer.
+    As an evolution of the the hugely successful VUPLAY SDK, this SDK has a number of key benefits:
+    ● Complete control over the entire AVPlayer lifecycle
+    ● Numerous optimisations to enable faster playback
+    ● Bitcode support
+    ● Support for building/linking against the simulators
+    The SDK is fully supported in both Objective-C and Swift applications. A demo application written in Swift is available on request.
+    Please contact support@vualto.com to request a download of the SDK.
+
+### REQUIREMENTS
+
+● Minimum iOS deployment target of 9.x or higher
+● Xcode 8.3.3
+● Swift 3.1
+
+### XCODE INTEGRATION
+
+1. Copy the vudrmFairPlay.framework into your application's project directory
+2. Navigate to your target and add vudrmFairPlay.framework to your ‘Embedded Binaries’. You should see that vudrmFairPlay.framework also now appears under ‘Linked Frameworks and Libraries’:
+
+### INFORMATION ABOUT APPLICATION TRANSPORT SECURITY (ATS)
+
+iOS/tvOS 9 introduces Application Transport Security (ATS) which restricts the use of insecure HTTP.
+In order to permit playback of content over insecure HTTP exemptions need to be added into your application's Info.plist. For example to disable ATS for any connection you would add the following into your application's Info.plist:
+<dict>
```

Comment:
Surround snippet in triple back-ticks and apply appropriate indentation

Reply:
any changes to this document will have to be done by Adam